technic/machines/other/constructor.lua

```diff
@@ -142,21 +142,38 @@ local function make_constructor(mark, length)
 			"technic_constructor_front_off.png"},
 		paramtype2 = "facedir",
 		groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-			mesecon = 2, technic_constructor = 1},
+			  mesecon = 2, technic_constructor = 1, axey=2, handy=1},
```

Comment:
Minor indent / style
```suggestion
			mesecon = 2, technic_constructor = 1, axey=2, handy=1},
```